### PR TITLE
Import single direction CNECs

### PR DESCRIPTION
**Pull request by phiedw**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
allows the use of single direction CNECs

**What is the current behavior?** *(You can also link to an open issue here)*
The getMinThreshold and getMaxThreshold functions act as if the CNEC threshold is in both directions.

**What is the new behavior (if this is a feature change)?**
The getMinThreshold and getMaxThreshold functions return different values depending on the direction of the threshold.

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java [hunk 1]
```diff
@@ -67,14 +67,20 @@ public PhysicalParameter getPhysicalParameter() {
 
     @Override
     public Optional<Double> getMinThreshold(Unit requestedUnit) throws SynchronizationException {
-        // TODO : handle cases where direction != Direction.BOTH
-        return Optional.of(-convert(getAbsoluteMax(), unit, requestedUnit));
+        if (direction == Direction.OUT) {
```
Comment (baptsegu) [inline data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java]: Do we really want to keep Direction.IN and Direction.OUT ?
I would prefer (and @terriervik too), Direction.ORIGIN_TO_EXTREMITY and Direction.EXTREMITY_TO_ORIGIN, or (but we are less fans) Direction.DIRECT and Direction.INDIRECT

File: data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java [hunk 2]
```diff
@@ -67,14 +67,20 @@ public PhysicalParameter getPhysicalParameter() {
...
+            return Optional.of(Double.NEGATIVE_INFINITY);
```
Comment (baptsegu) [inline data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java]: We should return Optional.empty() -> It means there is no Threhold limits and in that case there is no constraint in the optimisation problem (and we do not build a dummy constraint Flow >= -infinity).

File: data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java (hunk 1 above)
Comment (phiedw) [reply data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java]: modified to Direction.DIRECT and Direction.OPPOSITE

File: data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java (hunk 2 above)
Comment (phiedw) [reply data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/threshold/AbstractFlowThreshold.java]: switched to Optional.empty()
(modified a few Double.MIN_VALUE / Double.MAX_VALUE to Double.NEGATIVE_INFINITY / Double.POSITIVE_INFINITY)

**Pull request merged**